CIP-????/README.md

```diff
@@ -0,0 +1,113 @@
+---
+CIP: ?
+Title: Deterministic universal almost-unique Plutus Constructors
+Category: Plutus
+Status: Proposed
+Authors:
+    - Niels Mündler <niels@opshin.dev>
+Implementors: [Niels Mündler <niels@opshin.dev>]
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/608
+Created: 2023-10-20
+License: CC-BY-4.0
+---
+
+<!-- Existing categories:
+
+- Meta     | For meta-CIPs which typically serves another category or group of categories.
+- Wallets  | For standardisation across wallets (hardware, full-node or light).
+- Tokens   | About tokens (fungible or non-fungible) and minting policies in general.
+- Metadata | For proposals around metadata (on-chain or off-chain).
+- Tools    | A broad category for ecosystem tools not falling into any other category.
+- Plutus   | Changes or additions to Plutus
+- Ledger   | For proposals regarding the Cardano ledger (including Reward Sharing Schemes)
+- Catalyst | For proposals affecting Project Catalyst / the Jörmungandr project
+
+-->
+
+Note: I will use record / Plutus Data exchangibly throughout the document.
+
+## Abstract
+Plutus Constructor IDs are currently heavily focused around their origin in Haskell. They are usually used to distinguish different constructors of a single declared datatype.
```

Comment:
This section lacks an elaborate example, like an actual `Data` object. It took me a while to figure out what you mean by a constructor ID and I'm a Plutus developer.